plugins/kotlin-as-java/src/main/kotlin/transformToJava.kt

```diff
@@ -0,0 +1,26 @@
+package org.jetbrains.dokka.kotlinAsJava
+
+import org.jetbrains.dokka.kotlinAsJava.converters.asJava
+import org.jetbrains.dokka.kotlinAsJava.transformers.JvmNameDocumentableTransformer
+import org.jetbrains.dokka.model.DClasslike
+import org.jetbrains.dokka.model.DFunction
+import org.jetbrains.dokka.model.DPackage
+import org.jetbrains.dokka.model.DProperty
+import org.jetbrains.dokka.plugability.DokkaContext
+
+
+fun DPackage.transformToJava(context: DokkaContext): DPackage {
+    return JvmNameDocumentableTransformer().transform(this.asJava(), context)
+}
+
+fun DClasslike.transformToJava(context: DokkaContext): DClasslike {
+    return JvmNameDocumentableTransformer().transform(this.asJava(), context)
+}
```

Comment:
I also wonder if it makes sense to extract the creation of `JvmNameDocumentableTransformer` into a private lazy field, like
```
private val JVM_NAME_DOCUMENTABLE_TRANSFORMER by lazy { JvmNameDocumentableTransformer() }
```

I'd assume `transformToJava` methods will be called recursively and probably more than once, so it'd help reducing allocation rate and ease gc work.

Reply:
Or, better yet, I think `KotlinAsJavaPlugin#jvmNameTransformer` can be reused. This should also help avoid potential configuration problems in the future if the transformer implementation changes

Reply:
Recursive calls are not a problem, since these are `asJava` that go recursively and then `JvmNameDocumentableTransformer` is called only once. 

Resuing `jvmNameTransformer` as a property is also problematic since I cannot access `transform` method, I have access only to `invoke` method that is designed for `DModule` only.

The only solution here is to have lazy singleton of that transformer.